hapi/package.json

```diff
@@ -17,31 +17,32 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@hapi/boom": "^9.1.3",
-    "@hapi/hapi": "^20.1.5",
-    "axios": "^0.21.1",
+    "@hapi/boom": "^10.0.0",
+    "@hapi/hapi": "^20.2.2",
+    "axios": "^0.27.2",
     "eosjs": "^22.1.0",
     "eosjs-api": "^7.0.4",
-    "graphql": "^15.5.1",
-    "graphql-request": "^3.4.0",
+    "graphql": "^16.5.0",
+    "graphql-request": "^4.3.0",
     "http-status-codes": "^2.1.4",
     "joi": "^17.4.1",
-    "node-fetch": "^2.6.1"
+    "node": "^18.7.0",
+    "node-fetch": "^3.2.10"
   },
   "devDependencies": {
     "env-cmd": "^10.1.0",
-    "eslint": "^7.6.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-config-standard": "^16.0.2",
-    "eslint-plugin-import": "^2.22.1",
+    "eslint": "^8.21.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-standard": "^17.0.0",
+    "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "eslint-plugin-promise": "^4.3.1",
-    "husky": "4.3.8",
-    "lint-staged": "^10.5.4",
-    "nodemon": "^2.0.4",
-    "prettier": "^2.2.1",
-    "standard": "^16.0.3"
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^6.0.0",
+    "husky": "^8.0.1",
```

Comment:
Husky requires a new structure, please, check the [official documentation](https://typicode.github.io/husky/#/?id=install).